src/main/java/org/broadinstitute/consent/http/resources/DataRequestElectionResource.java

```diff
@@ -61,16 +61,13 @@ public Response createDataRequestElection(@Context UriInfo info, Election rec,
         URI uri;
         Election accessElection = null;
         try {
+            DataAccessRequest dar = darService.findByReferenceId(requestId);
+            boolean manualReview = dar.requiresManualReview();
             accessElection = api.createElection(rec, requestId, ElectionType.DATA_ACCESS);
-            List<Vote> votes;
+            List<Vote> votes = voteService.createVotes(accessElection, ElectionType.DATA_ACCESS, manualReview);
             //create RP election
-            if (!Objects.isNull(darApi.getField(requestId, DarConstants.RESTRICTION))) {
-                votes = voteService.createVotes(accessElection, ElectionType.DATA_ACCESS, false);
-                Election rpElection = api.createElection(rec, requestId, ElectionType.RP);
-                voteService.createVotes(rpElection, ElectionType.RP, false);
-            } else {
-                votes = voteService.createVotes(accessElection, ElectionType.DATA_ACCESS, true);
-            }
+            Election rpElection = api.createElection(rec, requestId, ElectionType.RP);
+            voteService.createVotes(rpElection, ElectionType.RP, false);
```

Comment:
This is the meat of the PR. The old logic would **_ONLY_** create an RP election if there was a `UseRestriction` object in the dar data. That is an unnecessary check, as ALL DARs will always have some form of`UseRestriction`, but not as a hard-coded field on the DAR since it is a calculable value from the existing questions on the DAR.